src/libraries/Microsoft.PowerFx.Json/Functions/JsonFunctionImpl.cs

```diff
@@ -287,6 +335,12 @@ public void Visit(TableValue tableValue)
 
                     foreach (DValue<RecordValue> row in tableValue.Rows)
                     {
+                        if (_checkCancellation(0))
+                        {
+                            _writer.WriteEndArray();
```

Comment:
>_writer.WriteEndArray(); [](http://example.com/codeflow?start=28&length=24)

Do we need to write the end array, since it will be ignored (as there is an error)?